lte/gateway/python/scripts/dhcp_helper_cli.py

```diff
@@ -0,0 +1,353 @@
+#!/usr/bin/env python3
+
+"""
+Copyright (C) 2022  The Magma Authors
```

Comment:
I think this license is wrong, should it not be:
```
"""
Copyright 2022 The Magma Authors.

This source code is licensed under the BSD-style license found in the
LICENSE file in the root directory of this source tree.

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
"""
```

Reply:
No, it has to GNU, that's why we're putting the functionality into a separate executable.

Reply:
Oh okay, then ignore these two comments, thanks! :)